verification/extendTCI.go

```diff
@@ -153,21 +161,3 @@ func TestExtendTciOnDerivedContexts(d TestDPEInstance, c DPEClient, t *testing.T
 		t.Errorf("[ERROR]: Child node's cumulative TCI %x, expected %x", childTcbInfo.Fwids[1].Digest, wantCumulativeTCI)
 	}
 }
-
-func verifyMeasurements(c DPEClient, t *testing.T, handle *ContextHandle, expectedCurrent []byte, expectedCumulative []byte) {
-	handle, tcbInfo, err := getTcbInfoForHandle(c, handle)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	// Check that the last TcbInfo current/cumulative are as expected
-	current := tcbInfo.Fwids[0].Digest
-	cumulative := tcbInfo.Fwids[1].Digest
-	if !bytes.Equal(current, expectedCurrent) {
-		t.Errorf("[ERROR]: Unexpected TCI_CURRENT digest, want %v but got %v", expectedCurrent, current)
-	}
-
-	if !bytes.Equal(cumulative, expectedCumulative) {
-		t.Errorf("[ERROR]: Unexpected cumulative TCI value, want %v but got %v", expectedCumulative, cumulative)
-	}
-}
```

Comment:
Made as a helper function not depending on DPEClient and Testing.t
Have moved into certs.go as it has a collection of all helper functions that operate on cert and cert extensions.